### DISCUSSION TITLE
See logs in headless mode

### DISCUSSION THREAD

**Discussion by farzadmf** [WhoIsSethDaniel/mason-tool-installer.nvim#54] [Q&A] [answered]
I have the following command to update the tools in the terminal, using `headless` mode:

```sh
nvim --headless +'autocmd User MasonToolsUpdateCompleted quitall' +'MasonToolsUpdateSync'
```

When I run this command, it runs and exists without no logs, _but_ a coupld of packages have issues to update, which can be seen if I don't run the `autocmd`:

```sh
nvim --headless +'MasonToolsUpdateSync'

pyright: updating to 1.1.370vtsls: updating to 0.2.5
pyright: failed to install
vtsls: failed to install
```

Is there a way I can see the logs in the first example?

**Comment by WhoIsSethDaniel** [comment, marked as answer]
This was answered in #53.
